tests/e2e/workloads/amq/test_amq_pod_respin.py

```diff
@@ -0,0 +1,106 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.framework.testlib import (
+    E2ETest, workloads, ignore_leftovers
+)
+from tests.helpers import default_storage_class
+from tests.disruption_helpers import Disruptions
+from ocs_ci.utility import templating
+from ocs_ci.ocs.resources.pod import get_all_pods
+
+
+log = logging.getLogger(__name__)
+
+
+def respin_amq_app_pod(kafka_namespace):
```

Comment:
Do you think it is a good idea to move this function in amq class so that it can be used in other tests as well?

Reply:
Since this function is more specific for one test case so defined here.

Reply:
Ack. If there are new test cases which requires to respin app pod then we can consider moving this function to amy class.